node/subsystem-types/src/messages.rs

```diff
@@ -604,6 +608,7 @@ impl ChainSelectionMessage {
 			ChainSelectionMessage::Approved(_) => None,
 			ChainSelectionMessage::Leaves(_) => None,
 			ChainSelectionMessage::BestLeafContaining(..) => None,
+			ChainSelectionMessage::DisputeConcludedAgainst(..) => None,
```

Comment:
This method always returns `None`? What am I missing here, haha. And doesn't this new variant contain a relay parent hash?

Reply:
This is a function that needs to be implemented for each message type the overseer deals with I think. Also, relay_parent() seems to be for providing context to information requests such as `RuntimeApiMessage::Request(hash)`. Since my message isn't asking the runtime for anything, I think it can safely return None.

Reply:
Crafted a PR to remove these implementations: https://github.com/paritytech/polkadot/pull/6525 . Looks like they are not really used.